src/i18n.js

```diff
@@ -14,6 +14,7 @@ const languageList = {
     "es-ES": "Español",
     "eu": "Euskara",
     "fa": "Farsi",
+    "ga": "Gaeilge",
```

Comment:
Duplicated. Maybe it is available in v2 already.

Reply:
Sorry ><, thanks!